deploy/lib/ml_rest.rb

```diff
@@ -114,35 +82,35 @@ def install_extensions(path)
         data.each_with_index do |d, i|
           file = open(d, "rb")
           contents = file.read
-
+          
```

Comment:
Your PR has a bunch of extraneous whitespace additions. If you set up your editor with [editorconfig](http://editorconfig.org/), it should pick up the .editorconfig file in this project and manage whitespace correctly. Please fix and update.